src/Core/Configuration/ConfigurationService.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EtoA\Core\Configuration;
+
+class ConfigurationService
```

Comment:
long term I would probably split this into two classes. A service that deals with loading and the XML stuff and a class that holds the configuration.

Reply:
Done 1b35a85